pkg/action/uninstall.go

```diff
@@ -71,6 +72,9 @@ func (u *Uninstall) Run(name string) (*release.UninstallReleaseResponse, error)
 
 	rels, err := u.cfg.Releases.History(name)
 	if err != nil {
+		if u.IgnoreNotFound {
```

Comment:
Is `notFound` the only error that can be returned by the release history command? Digging through the storage driver for secrets, it seems like there are multiple errors that could be returned. I think to be thorough this should check for a specific error condition before returning nil.

Additionally, I wonder if a debug log message would be appropriate here.

Reply:
Good point, I'll check.

Reply:
I've update the code to check for `ErrReleaseNotFound` explicitly.